client/src/components/game/phases/trade/ActiveTrade.vue

```diff
@@ -185,6 +187,14 @@ export default class ActiveTrade extends Vue {
 
   statusColor(type:string){
     let color = 'white';
+
+    if(this.involved == 1){
+      color = 'var(--new-space-orange)';
+    }
+    else if(this.involved == 0){
+      color = 'var(--trade-request-blue)';
```

Comment:
not sure that having a blue color is necessary. trades could just be orange if they're associated with the player (received or sent by the player)

also any trades colors that are being set should coordinate with mars log message colors, log colors can be changed to match trades or vice versa. but in this case, any color assignments should persist from trades to mars logs. there is a color map for mars log colors in `getters.ts`